Source/UltraleapTrackingCore/Private/LeapTrackingSettings.cpp

```diff
@@ -0,0 +1,21 @@
+/******************************************************************************
+ * Copyright (C) Ultraleap, Inc. 2011-2024.                                   *
+ *                                                                            *
+ * Use subject to the terms of the Apache License 2.0 available at            *
+ * http://www.apache.org/licenses/LICENSE-2.0, or another agreement           *
+ * between Ultraleap and you, your company or other organization.             *
+ ******************************************************************************/
+
+
+#include "LeapTrackingSettings.h"
+#include "LeapBlueprintFunctionLibrary.h"
+
+
+#if WITH_EDITOR
+void ULeapTrackingSettings::PostEditChangeProperty(FPropertyChangedEvent& PropertyChangedEvent)
+{
+	Super::PostEditChangeProperty(PropertyChangedEvent);
+	TArray<FString> Serials;
```

Comment:
Why do we pass an empty Serials array here. Is this just operating in default mode for the hints?

Reply:
Yes, this is for the default mode